test/specs/footer.ts

```diff
@@ -0,0 +1,114 @@
+import { expect } from "@wdio/globals";
+import axios from "axios";
+import MainPage from "../pageobjects/main.page.ts";
+import * as testData from "../data/testdata.json"
+
+
+describe("Rentzila", () => {
+  it("C214-Verify that all elements on the footer are displayed and all links are clickable", async () => {
```

Comment:
@Anastas11aZakharova This test case failed for me.

Reply:
As discussed  with Bohdan Alieinykov, the issue is in the modal pop up window that behaves randomly in WDIO. We tried multiple approaches to handle this, even copied wdio.conf and ts.config fully from a real project. Non of this worked unfortunately . 
Agreed to wait for new chrome and WDIO version to see if it becomes more stable.

Reply:
Passes every time for me. Can you please share the error message and error line?